transferui/src/test/java/com/hyperwallet/android/ui/transfer/viewmodel/ListTransferDestinationViewModelTest.java

```diff
@@ -86,36 +88,39 @@ public Object answer(InvocationOnMock invocation) {
                 callback.onTransferMethodListLoaded(null);
                 return callback;
             }
-        }).when(transferMethodRepository).loadTransferMethods(ArgumentMatchers.any(
+        }).when(mTransferMethodRepository).loadTransferMethods(ArgumentMatchers.any(
                 TransferMethodRepository.LoadTransferMethodListCallback.class));
 
         ListTransferDestinationViewModel viewModel = factory.create(ListTransferDestinationViewModel.class);
+        viewModel.init();
 
         assertThat(viewModel.getTransferDestinationList(), is(notNullValue()));
         assertThat(viewModel.getTransferDestinationList().getValue(), Matchers.<TransferMethod>hasSize(0));
     }
 
     @Test
     public void testGetTransferDestinationSection_returnsLiveData() {
-        assertThat(mModelToTest.getSelectedTransferDestination(), is(notNullValue()));
+        assertThat(mListTransferDestinationViewModel.getSelectedTransferDestination(), is(notNullValue()));
     }
 
     @Test
     public void testGetTransferDestinationError_returnsLiveData() {
-        assertThat(mModelToTest.getTransferDestinationError(), is(notNullValue()));
+        assertThat(mListTransferDestinationViewModel.getTransferDestinationError(), is(notNullValue()));
     }
 
     @Test
     public void testInitialization_callsRepository() {
```

Comment:
This test can be removed. We don't need to check that the repository was invoked here. The test 'testInit' already checks that loadTransferDestinationList() and we can add `verify(mTransferMethodRepository).loadTransferMethods(
                any(TransferMethodRepository.LoadTransferMethodListCallback.class));`
there

Reply:
Done